docs/querying/api.md

```diff
@@ -208,6 +208,10 @@ $ curl 'http://localhost:9090/api/v1/query_range?query=up&start=2015-07-01T20:10
 
 ## Querying metadata
 
+Prometheus offers a set of API endpoints to query metadata about series and their labels.
+
+NOTE: These API endpoints may return additional metadata for series outside of the selected time range, or for series whose samples have been deleted via the deletion API endpoint. The exact extent of additionally returned metadata is an implementation detail that may change in the future.
```

Comment:
It's not additional metadata, it's additional series.  I'd shorten it to that it may return series for which there is not a sample  within the given time range.

Reply:
Ok, that may be worded a bit confusingly, although of course I meant it like you. I'll adopt something along those lines.